inspectors/view/qml/notation/NotationInspectorView.qml

```diff
@@ -44,6 +44,7 @@ InspectorSectionView {
         width: parent.width
 
         columns: 2
+        columnSpacing: 12
```

Comment:
One is for column spacing of a grandparent widget, one is for minimum width of a grandchild widget. This is not C++, so no static classes or global variables or macros. It's indeed almost impossible. Unless @vpereverzev can think of something?

Reply:
No idea, I don't know that much of QML, but can singletons help here? Or <code>.pragma library</code>?

Reply:
Some possible approaches to resolving this issue are listed in https://wiki.qt.io/Qml_Styling. In addittion to what is listed there, it is possible to define a `.js` library defining style values to be used across the code (like in https://github.com/musescore/MuseScore/pull/5024/files#diff-3b8ee00d5e14695435aec00d5d3ec9c8) or defining a [context property](https://doc.qt.io/qt-5/qqmlcontext.html#details) with these values on C++ side. However I should admit that QML palettes still use a lot of magic constants, although some of them are packed into "styled" components set.

I guess if there is a goal to reduce values duplication there should be some sort of guideline on how these values should be defined. As I tried to show above, there are multiple ways to resolve this issue, and it would be good not to end up using them all at once :)

Reply:
@vpereverzev said it's OK, no need to avoid having the same numbers in two places.

Reply:
Even 3 places now it seems. I don't like this, these kind of magic numbers scattered all over the place

Reply:
This is a value for UI design, not for some hardcore functionality which always needs the value to be unified. If by some time we should change the column spacings to different values, all of the efforts simply for avoiding "magic numbers", if have been done, will be rendered useless.